remote-api/src/main/java/org/alfresco/rest/api/nodes/NodeRulesRelation.java

```diff
@@ -103,6 +105,14 @@ public Rule readById(String folderNodeId, String ruleSetId, Parameters parameter
         return rules.getRuleById(folderNodeId, ruleSetId, ruleId);
     }
 
+    @Override
+    public List<Rule> create(String folderNodeId, List<Rule> rules, Parameters parameters)
```

Comment:
To avoid confusion, maybe `List<Rule> rules` could be renamed to `List<Rule> ruleList`?
That way we could also get rid of `this` keyword in line 113.